www/src/data/sidebars/doc-links.yaml

```diff
@@ -149,16 +149,21 @@
         - title: Plugins
           link: /docs/plugins/
           items:
-            - title: How Plugins Work
-              link: /docs/how-plugins-work/
-            - title: Creating a Transformer Plugin
-              link: /docs/creating-a-transformer-plugin/
-            - title: Creating a Source Plugin
-              link: /docs/creating-a-source-plugin/
-            - title: Source Plugin Tutorial
-              link: /docs/source-plugin-tutorial/
             - title: Plugin Library
               link: /plugins/
+            - title: Creating Plugins
+              link: /docs/creating-plugins/
+              items:
+                - title: How Plugins Work
+                  link: /docs/how-plugins-work/
+                - title: Creating a Source Plugin
+                  link: /docs/creating-a-source-plugin/
+                - title: Creating a Transformer Plugin
+                  link: /docs/creating-a-transformer-plugin/
+                - title: Submit to Plugin Library
+                  link: /contributing/submit-to-plugin-library/
+                - title: Source Plugin Tutorial
```

Comment:
`Source Plugin Tutorial` is really similar to `Creating a Source Plugin`. Can we differentiate them any further, like `Source Plugin Tutorial: Pixabay`?

Reply:
Yeah that sounds great! I'll make that change.

Reply:
ok done!